.swiftlint.yml

```diff
@@ -0,0 +1,22 @@
+disabled_rules: # rule identifiers to exclude from running
+  - nesting
+# parameterized rules are first parameterized as a warning level, then error level.
+line_length:
+  - 200 # warning
+  - 300 # error
+function_body_length:
+  - 300
```

Comment:
It looks like this value is number of lines: https://github.com/realm/SwiftLint/blob/master/Rules.md#L8852

300 lines is a really really long function. I think the Android repo doesn't allow functions longer than 30 lines, but maybe that's a little too low. I'm not sure what the correct length would be for Swift, but how about something like 30 lines for a warning and 50 for an error?

Reply:
Agreed. I think this was needed for the Quick/Nimble BDD tests to get past swiftlint. Maybe it's better to add the Tests folder to the excludes list.

Reply:
Done